fxmanifest.lua

```diff
@@ -5,24 +5,29 @@ description 'QBX-MechanicJob'
 version '1.0.0'
 
 shared_scripts {
-	'@qb-core/shared/locale.lua',
-	'locales/en.lua', -- Edit this for your language.
-	'config.lua',
-	'@ox_lib/init.lua',
+    '@qb-core/shared/locale.lua',
+    'locales/es.lua',
```

Comment:
```suggestion
    'locales/en.lua',
```
Default language should be English